<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgba(255, 255, 255, 0);
	border: 1px solid gray;
	border-right: none;
	border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}
QLabel:hover{
	color: rgb(255, 93, 35);
	border-color: rgb(255, 93, 35);
}
QLabel:pressed{
	background-color: rgb(255, 93, 35);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>    姓名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="textName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: rgba(255, 255, 255, 0);
	border: 1px solid gray;
	border-left: none;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
	color: rgb(51, 51, 51);
}
QLineEdit:hover{
	color: rgb(255, 93, 35);
	border-color: rgb(255, 93, 35);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgba(255, 255, 255, 0);
	border: 1px solid gray;
	border-right: none;
	border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}
QLabel:hover{
	color: rgb(255, 93, 35);
	border-color: rgb(255, 93, 35);
}
QLabel:pressed{
	background-color: rgb(255, 93, 35);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>    客户名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="textUser">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: rgba(255, 255, 255, 0);
	border: 1px solid gray;
	border-left: none;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
	color: rgb(51, 51, 51);
}
QLineEdit:hover{
	color: rgb(255, 93, 35);
	border-color: rgb(255, 93, 35);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSure">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>270</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
	border: 1px solid gray;
	border-radius: 20px;
	font: 75 14pt &quot;微软雅黑&quot;;
}
QPushButton:hover{
	color: rgb(255, 93, 35);
	border-color: rgb(255, 93, 35);
}
QPushButton:pressed{
	background-color: rgb(255, 93, 35);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="LOGO">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>10</y>
     <width>135</width>
     <height>54</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
